Dockerfile

```diff
@@ -68,27 +86,33 @@ RUN ${CONDA_EXECUTABLE} env create -f ./danesfield/deployment/conda/conda_env.ym
     ${CONDA_EXECUTABLE} clean -tipsy
 
 # Install core3d-tf_ops package from kitware-danesfield / defaults
-RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && \
-      conda activate core3d && \
-      conda install -c kitware-danesfield -c kitware-danesfield-df -y core3d-tf_ops && \
-      conda clean -tipsy"]
+RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && conda activate core3d && pip install pip==20.0.1 && conda install -c kitware-danesfield -c kitware-danesfield-df -y core3d-tf_ops && conda clean -tipsy"]
```

Comment:
We usually break lines around 80 characters or so, this isn't a hard requirement but I think it does make things a bit easier to parse.